.env

```diff
@@ -0,0 +1,5 @@
+WEBDRIVERPATH=C:\path\to\gecko
+TWILIOTONUM=55555555
```

Comment:
Thanks for making these changes! One small thing: could the two example numbers be in the correct format? Ex. TWILIOTONUM=+12223334444

Reply:
Other than that, looks great! Thanks so much!